deploy/config/global_config.init.yaml

```diff
@@ -28,3 +29,8 @@ runtime:
     zone: .*
     image_id: xenial4x64a
     image_url: https://openpitrix.pek3a.qingstor.com/image/ubuntu.tar.gz
+  aws_provider:
+    api_server: ec2.us-east-2.amazonaws.com
+    zone: us-east-2
```

Comment:
The image id is different between different zone?

Reply:
Yes.